hyrax.gemspec

```diff
@@ -37,8 +37,7 @@ SUMMARY
   spec.add_dependency 'blacklight', '~> 6.14'
   spec.add_dependency 'blacklight-gallery', '~> 0.7'
   spec.add_dependency 'breadcrumbs_on_rails', '~> 3.0'
-  # Pin browse-everything to stable version
-  spec.add_dependency 'browse-everything', '< 0.16'
+  spec.add_dependency 'browse-everything', '< 2.0'
```

Comment:
I think this might need to be '>= 0.16', '< 2.0' since it looks like `Retriever#can_retrieve?` didn't appear until 0.16.0.